### PR TITLE
Reduce net consumption

### PR DESCRIPTION
**Pull request by alonSadan**
This PR tries to reduce the net consumption done by the ovs-bridge-marker. This PR introduces the same changes done in [CNI-bridge](https://github.com/kubevirt/bridge-marker/pull/35): 
- increasing the update interval
- use cache to keep track on the available bridges
- use a jitter to avoid network peeks.

### CODE REVIEW

File: cmd/marker/main.go [hunk 1]
```diff
@@ -74,11 +84,27 @@ func main() {
...
+		}
+
+		err := markerApp.Update(&markerCache)
 		if err != nil {
 			glog.Fatalf("Update failed: %v", err)
 		}
-		time.Sleep(time.Duration(*pollInterval) * time.Second)
-	}
+
+	}, time.Duration(*updateInterval)*time.Second, 1.2, true, wait.NeverStop)
```
Comment (phoracek) [inline cmd/marker/main.go:109]: Any chance you did not run `go fmt` on this (surprised about different `*` formatting)?

**Review by phoracek — COMMENTED** [review]
Minor nit. Overall looks good

File: cmd/marker/main.go (hunk 1 above)
Comment (alonSadan) [reply cmd/marker/main.go:109]: Actually I did. I also re-checked and it stays in tact. Would you like me to change that? (this kind of formatting might also be needed in [marker_test](https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/tests/marker_test.go#L64) )

Comment (phoracek) [reply cmd/marker/main.go:109]: If both are compatible, it's all good. Thanks for checking

**Comment by alonSadan** [comment]
> /lgtm
> /approve

Thank you Petr

**Pull request merged**